addons/account/tests/test_invoice_taxes.py

```diff
@@ -408,7 +414,7 @@ def test_misc_journal_entry_tax_tags_purchase(self):
 
         self.assertRecordValues(move.line_ids.sorted('balance'), [
             {'balance': -1100.0,    'tax_ids': [],              'tag_ids': [],                      'tax_base_amount': 0},
-            {'balance': 100.0,      'tax_ids': [],              'tag_ids': self.tax_tag_pos.ids,    'tax_base_amount': 1000},
+            {'balance': 100.0,      'tax_ids': [],              'tag_ids': self.tax_tag_pos.ids,    'tax_base_amount': 1000,    'tax_repartition_line_id': inv_tax_rep_ln.id},
```

Comment:
All expected values inside `assertRecordValues` should have the same keys

Reply:
That's un peu moisi, if you want my opinion :p I'm gonna fix that.